src/pages/SatelliteManifestPage/__tests__/SatelliteManifestPage.test.tsx

```diff
@@ -39,22 +52,113 @@ describe('Satellite Manifests Page', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('renders correctly with no results', () => {
+  it('renders loading when it has not received a response back', async () => {
     window.insights = {};
+
+    const { authenticateUser } = PlatformServices;
+
+    (authenticateUser as jest.Mock).mockReturnValue(
+      Promise.resolve({
+        identity: {
+          user: {
+            is_org_admin: true
+          }
+        }
+      })
+    );
+
     (useSatelliteManifests as jest.Mock).mockReturnValue({
       isLoading: false,
       data: []
     });
 
     const { container } = render(
-      <Provider store={init().getStore()}>
-        <Router>
-          <QueryClientProvider client={queryClient}>
-            <SatelliteManifestPage />
-          </QueryClientProvider>
-        </Router>
-      </Provider>
+      <Authentication>
+        <Provider store={init().getStore()}>
+          <Router>
+            <QueryClientProvider client={queryClient}>
+              <SatelliteManifestPage />
+            </QueryClientProvider>
+          </Router>
+        </Provider>
+      </Authentication>
```

Comment:
Can you pull this whole thing being rendered out into a variable or something, so you don't have to repeat the same stuff for each test?

Reply:
OK Updated that.